code/datums/mutations/touch.dm

```diff
@@ -49,3 +54,46 @@
 	else
 		to_chat(user,"<span class='warning'>The electricity doesn't seem to affect [target]...</span>")
 		return ..()
+
+#define HALFWAYCRITDEATH ((HEALTH_THRESHOLD_CRIT + HEALTH_THRESHOLD_DEAD) * 0.5)
+
+/obj/item/melee/touch_attack/shock/proc/defib(mob/living/carbon/target, mob/living/carbon/user)
```

Comment:
this is like all copied and pasted, don't do this especially for something as packed as defib code

Reply:
God dammit, you mean I have to put in EFFORT ?

Reply:
I'll see what I can do.